app/src/main/java/com/android/bookswap/MainActivity.kt

```diff
@@ -226,7 +233,13 @@ class MainActivity : ComponentActivity() {
           }
         }
         navigation(startDestination = C.Screen.USER_PROFILE, route = C.Route.USER_PROFILE) {
-          composable(C.Screen.USER_PROFILE) { UserProfile(photoStorage) }
+          composable(C.Screen.USER_PROFILE) {
+            UserProfile(
+                photoStorage,
+                topAppBar = { topAppBar("Your Profile") },
```

Comment:
Right, next time, I'll *fix* some code to be able to work more efficiently, *revert* the fix before making the PR, before creating  new task that *may or may not* be assigned, so that the code I *already fixed* can be fixed...
Doing the work twice isn't efficient at all, and delaying bug-fixing is not a good practice either.

An alternative method to that would be to branch out of the current feature branch, make a PR to merge the fix in the feature branch and finally have a PR to merge the feature and the fix to main.

Reply:
Already answered here : https://github.com/BookswapEPFL/Bookswap/pull/300#issuecomment-2526202093

Is something is blocking you from working, sure fix it.
But don't go into a file trying to solve every single issue, this is not the point of your task nor your PR and remove the whole point.
Adding a topAppBar and bottomAppBar is not something that was discussed with the team and certainly not something we all agree on. I really doubt that missing the "Your profile" at the top was something making you less productive.

We cannot have each of your PR starting to fix something that we didn't assign before when it's not related at all ! The point of having a SM and tasks is to have a general feeling of what people are going to work on.
You deciding that your task on geocoords should also include the unnecessary and probably unwanted need to add a top and bottom bar is not normal.